worlds/dark_souls_3/__init__.py

```diff
@@ -93,6 +93,8 @@ def generate_early(self):
             self.enabled_location_categories.add(DS3LocationCategory.HEALTH)
         if self.multiworld.enable_progressive_locations[self.player] == Toggle.option_true:
             self.enabled_location_categories.add(DS3LocationCategory.PROGRESSIVE_ITEM)
+        if self.multiworld.early_banner[self.player] == Toggle.option_true:
```

Comment:
Should this be conditional on MISC locations being randomized?

Reply:
i'm not dealing with anything other than the Banner for this PR. this PR's intentions were not to go any deeper than to solve one singular issue, and that is having a Sphere 1 banner across worlds.

I'd say if thats something that you would like to see when you start implementing your take on the client, that would be the time to create a seperate PR along with your changes.

Reply:
I mean for the banner specifically—since the banner location is only randomized _at all_ if `enable_misc_locations` is set, you'll be requesting early assignment for an item that's not in the item pool at all when it's not.

Reply:
I got what you mean now, corrected it to be under KEY locations, as that is where the banner resides, is under KEYS not MISC